src/main/java/org/sonatype/nexus/ci/iq/IqPolicyEvaluatorWorkflowStep.groovy

```diff
@@ -67,11 +67,21 @@ class IqPolicyEvaluatorWorkflowStep
     this.jobCredentialsId = jobCredentialsId
   }
 
+
+  @DataBoundSetter
+  public void setIqApplication(final String iqApplication) {
```

Comment:
What does the Pipeline Syntax Generator give you with the field of IqApplication type and the DataBoundSetter a String type.

Reply:
> What does the Pipeline Syntax Generator give you with the field of IqApplication type and the DataBoundSetter a String type.

Didn't work.  Went back to Object.  But instanceof failed code quality check. Had to compare class names.  What I really want to get working was Method Overloading.  But Jenkins seemed to get confused and not always work.